functionary/ui/templates/core/scheduled_task_list.html

```diff
@@ -0,0 +1,72 @@
+    <div>
+
+        <div class="columns is-centered">
+            <div class="column has-text-centered is-half">
+                {% if messages %}
+                {% for message in messages %}
+                <div class="notification is-warning">
+                    <button class="delete" onclick="return this.parentNode.remove();"></button>
+                    {{ message }}
+                </div>
+                {% endfor %}
+                {% endif %}
+            </div>
+        </div>
+
+        <div class="level-right">
+
+            <a href="{% url 'ui:new-scheduled-task' %}">
+                <button class="button level-item is-link m-5">
+                    <span class="icon is-small">
+                        <i class="fa fa-solid fa-plus"></i>
+                    </span>
+                    <span>
+                        Create New Scheduled Task
+                    </span>
+                </button>
+            </a>
+        </div>
+        <div class="table-container">
+            <table class="table is-hoverable is-fullwidth">
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Function</th>
+                        <th>Last Run</th>
+                        <th>Schedule</th>
+                        <th>Status</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for scheduled_task in object_list %}
+                    <tr>
+                        <td>
+                            <a href="{% url 'ui:scheduled-task-detail' scheduled_task.id %}">{{ scheduled_task.name }}</a>
+                        </td>
+                        <td>
+                            {% if scheduled_task.function.display_name %}
+                                <a href="{% url 'ui:function-detail' scheduled_task.function.id%}">{{ scheduled_task.function.display_name }} ({{ scheduled_task.function.name }})</a>
+                            {% else %}
+                                <a href="{% url 'ui:function-detail' scheduled_task.function.id%}">{{ scheduled_task.function.name }}</a>
+                            {% endif %}
+                        </td>
+                        <td>
+                            {% if scheduled_task.most_recent_task is None %}
+                                Not run yet
+                            {% else %}
+                                <a href="{% url 'ui:task-detail' scheduled_task.most_recent_task.id %}">{{ scheduled_task.most_recent_task.created_at }}</a>
+                            {% endif %}
+                        </td>
+                        <td>{{ scheduled_task.periodic_task.crontab }}</td>
+                        <td>{{ scheduled_task.status }}</td>
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        </div>
+    </div>
+    <script src="{% static 'js/scheduled-task-list.js' %}" defer></script>
```

Comment:
This appears to be referencing a script that no longer exists.

Reply:
Fixed. Was a remnant from when I was going to do modals for the forms